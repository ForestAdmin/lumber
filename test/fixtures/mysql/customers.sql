create table customers (
  id int unsigned auto_increment primary key,
  name varchar(255) not null,
  description text null,
  is_active tinyint(1) default 1 not null,
  paying bit(1) default null,
  created_at date not null,
  updated_at date null
);
