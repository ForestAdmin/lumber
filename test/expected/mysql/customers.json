{
  "fields": [
    {
      "name": "name",
      "nameColumn": "name",
      "type": "STRING",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "description",
      "nameColumn": "description",
      "type": "STRING",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "isActive",
      "nameColumn": "is_active",
      "type": "INTEGER",
      "primaryKey": false,
      "defaultValue": "1"
    },
    {
      "name": "createdAt",
      "nameColumn": "created_at",
      "type": "DATE",
      "primaryKey": false,
      "defaultValue": null
    },
    {
      "name": "updatedAt",
      "nameColumn": "updated_at",
      "type": "DATE",
      "primaryKey": false,
      "defaultValue": null
    }
  ],
  "references": [],
  "primaryKeys": [
    "id"
  ],
  "options": {
    "underscored": true,
    "timestamps": true,
    "hasIdColumn": true,
    "hasPrimaryKeys": true
  }
}
